hashes/README.md

```diff
@@ -13,6 +13,17 @@ since these are needed to display hashes anway.
 
 This library should always compile with any combination of features on **Rust 1.48.0**.
 
+However, if you want to build this library using the MSRV toolchain you will may need to pin a
```

Comment:
"you will may" I think you meant "may" since not everyone depends on `serde`.

Reply:
Thanks man, fixed.